ArduPlane/ArduPlane.cpp

```diff
@@ -321,11 +321,18 @@ void Plane::efi_update(void)
 #endif
 }
 
+#if AP_AIRSPEED_AUTOCAL_ENABLE
```

Comment:
This doesn't actually work unless you also comment out the line in the header and the scheduler table. As it stands this would simply fail to build, and hasn't improved anything.

Reply:
You should also be surronding the function declaration in `Plane.h`